### DISCUSSION TITLE
Fan Control by Rem0o

### DISCUSSION THREAD

**Discussion by TheVoork** [ChrisTitusTech/winutil#1588] [Ideas]
I request that this app be added to the list in ChrisTitusTech Winutil
Fan Control is an awesome app that allows you to control your fans in various ways
https://getfancontrol.com/
https://github.com/Rem0o/FanControl.Releases

Also,
Belarc Advisor is a good idea
https://www.belarc.com/products/belarc-advisor
Thanks

**Comment by Fezzy976** [comment]
I second this too!!! Fan Control is an amazing piece of software!

**Comment by og-mrk** [comment]
This app has been added in a Pull Request (here's [a link to it](https://github.com/ChrisTitusTech/winutil/pull/1865)), in that PR, you'll notice [a commit](https://github.com/ChrisTitusTech/winutil/pull/1865/commits/864892e8d22488a74941f33aba380c19eecce684) which adds the `Fan Control` Program by Rem0o.
